plugins/rebalance/utils.go

```diff
@@ -0,0 +1,108 @@
+package rebalance
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/gluster/glusterd2/glusterd2/store"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	rebalancePrefix string = "rebalance/"
+)
+
+// TimeVal represents the time value
+type TimeVal struct {
+	TVsec  uint64
+	TVusec uint64
+}
+
+var (
+	hash uint64
+)
+
+func setCommitHash() uint64 {
+
+	/*
+	   We need a commit hash that won't conflict with others we might have
+	   set, or zero which is the implicit value if we never have.  Using
+	   seconds<<3 like this ensures that we'll only get a collision if two
+	   consecutive rebalances are separated by exactly 2^29 seconds - about
+	   17 years - and even then there's only a 1/8 chance of a collision in
+	   the low order bits.  It's far more likely that this code will have
+	   changed completely by then.  If not, call me in 2031.
+	   P.S. Time zone changes?  Yeah, right.
+	*/
+
+	tv := new(TimeVal)
+	t := time.Now()
+	tv.TVsec = uint64(t.Day() * t.Hour() * t.Minute() * t.Second())
```

Comment:
Are you expecting to get the number of unix seconds here? If so this is not correct. You can use `t.Unix()` or `t.UnixNano()` to get more granular values.

Reply:
Done